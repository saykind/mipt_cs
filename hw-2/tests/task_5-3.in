3
1 0 -1
3
1000000 2000000 3000000
