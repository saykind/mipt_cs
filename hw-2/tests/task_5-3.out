0 6000000.000000